app/src/main/res/values/colors.xml

```diff
@@ -44,6 +44,8 @@
   <color name="color_primary_dark_wtm">#3D9B86</color>
   <color name="color_primary_ioextended">#50B8D0</color>
   <color name="color_primary_dark_ioextended">#30A1BC</color>
+  <color name="color_primary_gcpnext">#E74235</color>
```

Comment:
No, there are my choices :) I checked colors of WTM and Devfest. They are not same with Hub. Are there any way to find the darker and lighter ones or what should we do to find right colors?